docs/io/ascii/write.rst

```diff
@@ -231,3 +231,181 @@ details.
+and ``dec`` components are divided into their ``hour``-``min``-``sec``
+and ``deg``-``arcmin``-``arcsec`` components respectively. Whereas, columns with
+``SkyCoord`` objects in the ``Galactic`` or any of the ``Ecliptic`` frames are divided
+into their latitude(``ELAT``/``GLAT``) and longitude components (``ELON``/``GLAT``) only.
+The following example illustrates this.
+
+  >>> from astropy.coordinates import SkyCoord
+  >>> table = Table()
+  >>> table['object'] = ['ASASSN-15lh']
+  >>> table['coord'] = SkyCoord.from_name('ASASSN-15lh')  # doctest: +REMOTE_DATA
+  >>> table.write('coord_cols.dat', format='ascii.cds')   # doctest: +SKIP
+  >>> table['coord'] = table['coord'].geocentrictrueecliptic  # doctest: +REMOTE_DATA
+  >>> table.write('ecliptic_cols.dat', format='ascii.cds')    # doctest: +SKIP
+
+The original table remains accessible as such, while the file is written from a
+modified copy of the table. Thus, the contents of ``coords_cols.dat`` will be::
+
+  Title:
+  Authors:
+  Table:
+  ================================================================================
+  Byte-by-byte Description of file: coords_cols.dat
+  --------------------------------------------------------------------------------
+   Bytes Format Units  Label     Explanations
+  --------------------------------------------------------------------------------
+   1-11  A11    ---    obj     Description of obj  
+  13-16  F4.1   h      RAh     Right Ascension (hour)  
+  18-20  F3.1   min    RAm     Right Ascension (minute)
+  22-39  F18.15 s      RAs     Right Ascension (second)
+     41  A1     ---    DE-     Sign of Declination     
+  42-45  F5.1   deg    DEd     Declination (degree)    
+  47-50  F4.1   arcmin DEm     Declination (arcmin)    
+  52-67  F16.13 arcsec DEs     Declination (arcsec)    
+  --------------------------------------------------------------------------------
+  Notes:
+  --------------------------------------------------------------------------------
+  ASASSN-15lh 22.0 2.0 15.450000000007265 -61.0 39.0 34.5999960000006
+
+And the file ``ecliptic_cols.dat`` will look like::
+
+  Title:
+  Authors:
+  Table:
+  ================================================================================
+  Byte-by-byte Description of file: ecliptic_cols.dat
+  --------------------------------------------------------------------------------
+   Bytes Format Units  Label     Explanations
+  --------------------------------------------------------------------------------
+   1-11  A11    ---    obj     Description of obj                         
+  13-29  F17.13 deg    ELON    Ecliptic Longitude (geocentrictrueecliptic)
+  31-48  F18.14 deg    ELAT    Ecliptic Latitude (geocentrictrueecliptic) 
+  --------------------------------------------------------------------------------
+  Notes:
+  --------------------------------------------------------------------------------
+  ASASSN-15lh 306.2242086500961 -45.62178985082456
+
+..
+  EXAMPLE END
```

Comment:
Would it make sense to reduce this to just two examples e.g. the remove the first two and start with a table that already has units and coordinates? The output is pretty long and our purpose here is to explain how to use it, not how the format looks (that is done in the linked CDS aad MRT websites). So, I do think we need to show an example of "your coordinate columns will be broken up automatically", but I also think that we don't have to explain every step.

On the other hand, I suggest adding an example of what you said in https://github.com/Suyog7130 :
Something like "MRT and CDS have specific naming conventions for columns (link to website). Most of those the CDS writer cannot enforce because it does not know what the data inside the columns means. For example, if a column contains the mean error for the data in a column named "label", then the column should be names "e_label". Since astropy cannot know how columns in the table are related, it is up to the user to use `Table.rename_colums` to appropriately rename any columns before write the MRT/CDS table." You could add an example for that, which will have lines like
```
[... set up table here ...]
outtab = tab.copy()  # so that changes don't affect the original table
outtab.rename_column('error', 'e_flux')
# re-order so that columns have good order
outtab = outtab['coord', 'name', 'flux', 'e_flux']  # set up your original table just that error and flux are not next to each other
# Or leave out the re-orderting. Not sure if it is useful as an example here. What do you think?
outtabl.write(...)

Reply:
Well, the error and other related columns are often put next to each other. So, I suppose, it would be good to add that.